website/src/app/kb/user-guides/linux-client/readme.mdx

```diff
@@ -35,17 +35,18 @@ Once you have a token, you can start the Linux Client using the following
 command:
 
 ```
-sudo ./linux-client-x64 <TOKEN>
+sudo FIREZONE_TOKEN=<TOKEN> ./linux-client-x64
 ```
 
 Set some environment variables to configure it:
 
 ```
-FIREZONE_NAME="Development Webserver"
-FIREZONE_ID="some unique identifier"
-DNS_CONTROL="systemd-resolved" # or "etc-resolv-conf"
-LOG_DIR="./"
-sudo -E ./linux-client-x64 <TOKEN>
+export FIREZONE_NAME="Development Webserver"
+export FIREZONE_ID="some unique identifier"
+export FIREZONE_TOKEN=<TOKEN>
+export DNS_CONTROL="systemd-resolved" # or "etc-resolv-conf"
+export LOG_DIR="./"
+sudo -E ./linux-client-x64
```

Comment:
Passing the token as an env var should be slightly better, but since headless clients must store it on disk either way, we should encourage users to just leave it on disk.

And I don't think this works without `export`, in interactive shells. In scripts it probably does? Brian and I both didn't remember.

Reply:
Yeah that sounds about right